include/revng/Model/ABI.h

```diff
@@ -295,15 +311,24 @@ inline constexpr llvm::StringRef getDescription(model::ABI::Values V) {
     return "32-bit Microsoft x86 abi that was intended to "
            "mimic 32-bit SystemV x86 abi but has minor "
            "differences";
+  case model::ABI::Microsoft_x86_cdecl_gcc:
+    return "Turns out gcc's version of `cdecl` is subtly different from "
+           "the original";
```

Comment:
1. This is a comment that is specifically used by the CLI tools (the ones that list all the ABIs we support), it makes a lot more sense in context - where it comes up right after the relevant msvc abi.

```yaml
Tool options:

  --abi=<value>                  - Specifies the default ABI of the binary.
    =Invalid                     -   Unknown and/or unsupported ABI
    # . . .
    =Microsoft_x86_cdecl         -   32-bit Microsoft x86 abi that was intended to mimic 32-bit SystemV x86 abi but has minor differences
    =Microsoft_x86_cdecl_gcc     -   Turns out gcc's version of `cdecl` is subtly different from the original
```

2. This predates YAML. But, IMO, since we plan to get rid of this entire enum eventually, the problem with duplicated descriptions will go away on its own.

Reply:
I'd go for something like `32-bit Microsoft x86 abi as implemented in GCC (subtly different from the MSVC one)`.

Reply:
Done.